docs/en/observability/logs-stream.asciidoc

```diff
@@ -143,10 +143,220 @@ If you're not seeing your log files in {kib}, check the following in the `elasti
 If you're still running into issues, see {fleet-guide}/fleet-troubleshooting.html[{agent} troubleshooting] and {fleet-guide}/elastic-agent-configuration.html[Configure standalone Elastic Agents].
 
 [discrete]
-[[logs-stream-whats-next]]
-== What's next?
+[[logs-stream-enhance-logs]]
+== Get the most out of your log data
 
-For more information on deploying and managing logs in Elastic Observability, see the following links:
+This section shows you how to get the most out of your log data after sending it to {es} by extracting structured fields from your unstructured log data. Extracting structured fields makes it easier to search, analyze, and filter your log data. 
 
-- The <<logs-checklist>> consolidates links to documentation on sending log data, configuring logs, and analyzing logs.
-- <<monitor-logs>> has information on visualizing and analyzing logs.
\ No newline at end of file
+Let's look at the following log example:
+
+[source,log]
+----
+2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
+----
+
+Add this log to {es} using the following command in *Dev Tools*, found in your deployment in the left navigation under *Management*:
+
+[source,console]
+----
+POST logs-generic-default/_doc
+{
+  "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%."
+}
+----
+
+Use the following command to look at the document's details:
+
+[source,console]
+----
+GET /<index-name>/_doc/<document-id>
```

Comment:
@ruflin I'm not sure this is the best way to do this. Do you have a better suggestion here?

Reply:
I would use the search command `GET /logs-generic-default/_search`. It will now only contain 1 doc and works even if the user does not know the doc id.